tests/components/zwave_js/test_discovery.py

```diff
@@ -240,6 +240,37 @@ async def test_shelly_001p10_disabled_entities(
     assert state
 
 
+async def test_qubino_flush_shutter_disabled_entities(
+    hass: HomeAssistant,
+    entity_registry: er.EntityRegistry,
+    client,
+    qubino_flush_shutter,
+    integration,
+) -> None:
+    """Test that Qubino Flush Shutter entity created by endpoint 2 is disabled."""
+    entity_ids = [
+        "cover.flush_shutter_2",
+    ]
+    for entity_id in entity_ids:
+        state = hass.states.get(entity_id)
+        assert state is None
+        entry = entity_registry.async_get(entity_id)
+        assert entry
+        assert entry.disabled
+        assert entry.disabled_by is er.RegistryEntryDisabler.INTEGRATION
+
+        # Test enabling entity
+        updated_entry = entity_registry.async_update_entity(
+            entry.entity_id, disabled_by=None
+        )
+        assert updated_entry != entry
+        assert updated_entry.disabled is False
```

Comment:
We need to reload the integration or move time forward after enabling an entity to actually have it be created and added.